imageprocessor/views.py

```diff
@@ -27,8 +29,9 @@ def classify(request):
 			image = Image.open(image_file)
 			context['tags'] = detect(image)
 			return render(request, 'output.html', context)
-		except OSError as err:
-			context['form'] = form
-			return render(request, 'input.html', context)
+		except ValueError:
+			context['errors'] = True
+		except OSError:
+			context['errors'] = True
```

Comment:
may want to look into using django messages instead of hardcoding the error message in the template.
https://docs.djangoproject.com/en/2.1/ref/contrib/messages/